src/web/app/pages-instructor/instructor-search-page/instructor-search-page.component.html

```diff
@@ -1,4 +1,6 @@
-<tm-instructor-search-bar (searched)="search($event)" [searchKey]="searchKey"></tm-instructor-search-bar>
+<tm-instructor-search-bar (searched)="search($event)" [searchParams]="searchParams"></tm-instructor-search-bar>
```

Comment:
I would suggest to use two-way data binding here `[(searchParams)]="searchParams"`